apps/meteor/app/api/server/v1/users.ts

```diff
@@ -334,7 +343,7 @@ API.v1.addRoute(
 			const user = await getUserFromParams(this.bodyParams);
 			const { confirmRelinquish = false } = this.bodyParams;
 
-			await Meteor.callAsync('deleteUser', user._id, confirmRelinquish);
+			await executeDeleteUser(this.userId, user._id, confirmRelinquish);
```

Comment:
I think we should actually call `deleteUser` here:

https://github.com/RocketChat/Rocket.Chat/blob/bbf12a1ad815e8689d62fa58c5538a963ebc4396/apps/meteor/app/lib/server/functions/deleteUser.ts#L34